sympy/printing/str.py

```diff
@@ -174,18 +174,14 @@ def _xab_tostr(xab):
         return 'Integral(%s, %s)' % (self._print(expr.function), L)
 
     def _print_Interval(self, i):
-        if i.left_open:
-            left = '('
+        fin =  'Interval(%s)'
```

Comment:
two spaces here, but it's not important. We usually try to keep SymPy code style as uniform as possible.